### ISSUE TITLE
Latest breadcrumb image

### ISSUE THREAD

**Issue by JanezCim** [UbiquityRobotics/breadcrumb_learn#27]
Hey, since we have sent out a couple of units of breadcrumb. Can you please have one latest working image which i can link into the tutorials? Thanks

**Comment by tp4348** [comment]
I could prepare it in the following week. @mbostic if you have time this week you can do it.

**Comment by anfederman** [comment]
I think the 12-07 image is Firefox, not Chrome, so make sure the official
image has Chrome and works with the LS lidar. The Ethernet port in Ubuntu
20 is set via Netplan, and needs to be checked

**Comment by JanezCim** [comment]
Id suggest we create a separate page on the breadcrumb learn titled "Breadcrumb Raspberry Pi images" or something similar and have it look something like the default ur download page: https://downloads.ubiquityrobotics.com/pi.html with old images and tutorial how to flash. Ideally i'd also really like to see the changelog for the images somewhere.

**Comment by mbostic** [comment]
Sadly I didn't have time today to create the image. But I did manage to make the exploration work at the end.

**Comment by JanezCim** [comment]
@mbostic is the lastest image ready to be linked?

**Comment by mbostic** [comment]
@JanezCim I don't think it is. It has a serious problem when LS lidar is connected and you are switching the WiFi.
- https://github.com/UbiquityRobotics/breadcrumb/issues/349
- https://github.com/UbiquityRobotics/breadcrumb/issues/339

@tp4348 what do you think? Is it ready?

**Comment by mbostic** [comment]
@JanezCim the issue https://github.com/UbiquityRobotics/breadcrumb/issues/349 was resolved by adding added the line `deny-interfaces=eth0` in file `/etc/avahi/avahi-daemon.conf` under the "server" part of the file (more details are here https://github.com/UbiquityRobotics/breadcrumb/issues/339), so I think it is almost time to create the new image. The only thing that needs to be done before is merging of this PR with some touchscreen improvements: https://github.com/UbiquityRobotics/breadcrumb/pull/355. Can you review the PR?

**Comment by anfederman** [comment]
Also in the after initial boot to make sure breadcrumb application comes
up. I am assuming the breadcrumb image is built from the default Noetic
image, and is a choice on initial configuration of:

Plain Magni (camera,sonar,lidar)
Breadcrumb (sensor options)
EZmap (sensor options)

After that, there is a config setup switch that allows reconfiguration and
the ability to purchase/download additional items.
